### DISCUSSION TITLE
在已连接US状态下有些网址访问不了，反而SG可以，使用困难，请求退款

### DISCUSSION THREAD

**Discussion by lizihui12345678** [getlantern/lantern-forum-cn#793] [General]
![image](https://github.com/user-attachments/assets/fb91c273-9de5-4d4d-83fc-2d0809850226)
还记得这个案例吗，我至今没有收到退款，请问什么时候可以收到呢？ @Mike777Khoo

**Comment by MaxMKMax** [comment]
@lizihui12345678 您好. 好的, 我帮您查看退款状态. 等待支付团队回复我.
